### ISSUE TITLE
10-7959c: update copy - Medicare section 

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#83660]
As a user on VA.gov, I want to fill out the OHI form with my information so that I can report any changes to my other health insurance.

Page elements:
- breadcrumb
- header and subhead
- progress indicator
- footer

Content pages:
- Medicare status (yes, no)
   - If yes: 
     - then ask Medicare coverage (Medicare plan options)
     - If yes to Medicare Advantage or Other answer options)
        - then show Medicare pharmacy benefits question (yes, no)
   - If no: user proceeds to Health insurance information section 
- Medicare carrier A (name and effective date)
- Medicare carrier B (name and effective date)
- file upload screen
- Medicare carrier D (name and effective date)
  - If yes: Medicare D carrier screen
    - file upload screen
  - If no: Health insurance information section

Figma link: https://www.figma.com/design/sSPoS7R9HE6ZlxRESRy2ms/10-7959c-Form?node-id=1155-73969&t=Y8cWvTGeyw2lKO6X-0 (sections noted with "wip" in yellow are not yet final)

Acceptance criteria:
  - Given the user is on VA.gov and is filling out the OHI form
  - When the user gets to the Medicare section
  - Then the user can update the information and the information appears in the PDF attachment of the form

**Comment by michaelclement** [comment]
Had a couple typos - fix is in[ this second PR](https://github.com/department-of-veterans-affairs/vets-website/pull/30192). Waiting on:
- [X] Team approval
- [ ] Platform approval

**Comment by michaelclement** [comment]
@marywang2 This has been merged and should be on staging shortly. Thanks!

**Comment by marywang2** [comment]
Thanks @michaelclement! Some comments from me and @jamiefiore:
- File upload decrease margin between the "Upload Medicare card and Upload file button to 8px> This appears on all file upload screens
- Make sure there is 8px of margin between the question and the hint text below it. It looks like this needs to be fixed throughout the form on the questions
- add period after 'fax' on medicare a/b file upload screen: https://staging.va.gov/health-care/champva/other-insurance-form-10-7959c/medicare-ab-upload.
- add spacing between line for medicare d file upload screen: https://staging.va.gov/health-care/champva/other-insurance-form-10-7959c/medicare-d-upload
- remove "Medicare additional comments" for now (we can discuss this one)

**Comment by michaelclement** [comment]
Work has been done in [this new PR](https://github.com/department-of-veterans-affairs/vets-website/pull/30236):
- [X] Decrease margin above upload button
- [ ] 8px margin above hints (this is being addressed in [this other PR](https://github.com/department-of-veterans-affairs/vets-website/pull/30234), but it's not final yet)
- [X] Period after fax on medicare upload screen
- [X] Spacing between lines on medicare D upload screen
- [X] Remove medicare comment box

**Comment by michaelclement** [comment]
Blocked until I get platform review

**Comment by michaelclement** [comment]
@marywang2 Just merged to staging, should be visible soon and then ready for review. Thanks!

**Comment by marywang2** [comment]
Thanks!
- Is there a right margin on the first Medicare status [page](https://staging.va.gov/health-care/champva/other-insurance-form-10-7959c/medicare-ab-status)
- update text to: "on the front..." https://staging.va.gov/health-care/champva/other-insurance-form-10-7959c/medicare-pharmacy
- remove "For example: January 19 2000" above month/date: https://staging.va.gov/health-care/champva/other-insurance-form-10-7959c/medicare-a-carrier
- remove "For example: January 19 2000" above month/date: https://staging.va.gov/health-care/champva/other-insurance-form-10-7959c/medicare-b-carrier
- remove extra space below "Medicare PACE card" (if there's extra?): https://staging.va.gov/health-care/champva/other-insurance-form-10-7959c/medicare-ab-upload
- remove "For example: January 19 2000" for Medicare D: https://staging.va.gov/health-care/champva/other-insurance-form-10-7959c/medicare-d-carrier

**Comment by michaelclement** [comment]
@marywang2 - all the requested changes from this ticket and the health insurance tickets ([1](https://app.zenhub.com/workspaces/ivc-forms-652da2d3f0ae4c0016bfb109/issues/gh/department-of-veterans-affairs/va.gov-team/83658) and [2](https://app.zenhub.com/workspaces/ivc-forms-652da2d3f0ae4c0016bfb109/issues/gh/department-of-veterans-affairs/va.gov-team/83948)) have been added in [this PR](@marywang2 This PR includes all requested changes for Medicare and Health insurance with the exception of margin changes) with the exception of the following:
- Margin changes. This is because we're trying to stick with the standard sizes used in current standard coded implementations across production
- Changes to the validation/error messages when tabbing through a date field. This is because date fields are composed of three separate fields, so tabbing through them triggers validation as intended.


Currently waiting on:
- [ ] Team approval
- [ ] Platform approval

**Comment by michaelclement** [comment]
@marywang2 This should be ready for you to take a look at. Thanks!
